### ISSUE TITLE
UVPD

### ISSUE THREAD

**Issue by pavel-shliaha** [sgibb/topdownr#67]
we now have UVPD fragmentation capacity and UVPD data.  UVPD takes 1 specific paramter:

"UVPDTime"

so we now have ETD, CID and HCD  columns, could you please also add UVPD. Also this has to be reflected in fragmentation maps of course, UVPD should get its own panel

the new data is in:

\Google Drive\example_files_for_topDown\new_dataset\2017_07_03_CA\UVPD

**Comment by sgibb** [comment]
Currently I set the `UvpdActivation` column as follows:
https://github.com/sgibb/topdownr/blob/7a3470d36e058403f9a4091f294a2f50e68e7e03/R/import.R#L164-L165

Should I use the `UvpdTime` column instead? (the above is working for the example files you gave me, Energy1 and UVPDTime are identical)

We have to provide the UVPD option in `writeMethodXmls` as well. I guess it is an `ms1Settings`. Do you have any information which nodes we have to put into the xml file?

**Comment by pavel-shliaha** [comment]
there is an error when trying to load files 

Reading 61 header information from file CA_UVPD_908_2.txt
Error in rbind(deparse.level, ...) : 
  numbers of columns of arguments do not match

I actually saw this error when I tried to trick R into thinking UVPD was CID.

This is probably related to different number of columns in the scanheadsman files.

> UVPD <- lapply (SH[grep ("UVPD", SH)], read.csv)
> sapply (UVPD, ncol)
[1] 67 67 67 67 67 67
> 
> nonUVPD <- lapply (SH[grep ("UVPD", SH, invert = TRUE)], read.csv)
> sapply (nonUVPD, ncol)
 [1] 79 79 79 79 79 79 79 79 79 79 79 79 79 79 79 79 79 79 79 79

I imagine this creates issues when you try to run rbind() for the coldata. I think the only suggestion is to have an argument that either preserves all columns (in this case empty columns need to be created) or just the common ones. This is conceptually the same as in merge function merge(all = TRUE)
